cmd/neofs-node/config.go

```diff
@@ -88,27 +80,45 @@ const notificationHandlerPoolSize = 10
 // values. It should not store any application helpers structs (pointers to shared
 // structs).
 // It must not be used concurrently.
+//
+// REFACTOR IS NOT FINISHED.
```

Comment:
unless it refers some issue or explain how the code should be maintained until the refactor is completed, this comment doesn't help anyone

Reply:
> this comment doesn't help anyone

IMO, it conflicts with the description: `stores component-specific configuration`, but the most of the config sections are omitted here

I added links to the original issue about SIGHUP and the original PR that does it, is it ok?

Reply:
understandable to me, but i'd appreciate issue in form `have this -> do this to got this`, pls fill the issue it async